nginx/src/css/sidebar.css

```diff
@@ -1,37 +1,127 @@
-#sidebar-wrapper {
-	position: fixed;
-	top: 0;
-	left: 0;
+#sidebar {
+	box-sizing: border-box;
 	height: 100vh;
 	width: 250px;
-	border-radius: 0 5px 5px 0;
-	transition: all 0.3s ease;
-	z-index: 1000;
-	text-align: center;
+	padding: 5px 1em;
+	background-color: var(--base-clr);
+	border-right: 1px solid var(--line-clr);
+
+	position: sticky;
+	top: 0;
+	align-self: start;
+	transition: 300ms ease-in-out;
+	overflow: hidden;
+	text-wrap: nowrap;
+}
+
+#sidebar.close {
+	padding: 5px;
+	width: 60px;
 }
 
-#sidebar-wrapper.toggled {
-	margin-left: -250px;
+#sidebar ul {
+	margin-left: -2.6rem;
+	list-style: none;
 }
 
-#sidebar-wrapper .sidebar-heading {
-	padding: 1rem;
-	font-size: 1.5rem;
+#sidebar>ul>li:first-child {
+	display: flex;
+	justify-content: flex-end;
+	margin-bottom: 16px;
+
+	.logo {
```

Comment:
not valid css

Reply:
Addressed in : [d24d348](https://github.com/TheSecretOrganization/ft_transcendence/pull/201/commits/d24d34875c8598f849f6a6c2f6e5f2b83ab06290)